### DISCUSSION TITLE
Magnisketch Drone Control

### DISCUSSION THREAD

**Discussion by aatishg01** [bitcraze/discussions#1696] [Show and tell]
My team and I used a drone to create sketches on a magnetic drawing board. We used MPC to create precomputed trajectories and gave the full states it produced to low level commander. 
Github: https://github.com/scott-wade/crazyflie_sketcher
Paper: https://arxiv.org/abs/2412.10670
Youtube video: https://youtu.be/jex1lPUhJAU
![Screenshot 2024-12-12 183633](https://github.com/user-attachments/assets/f74f76d2-f06c-4699-8a41-d6ba5b486e76)

**Comment by Bananazooka** [comment]
Hello, 
Thanks for sharing this, it looks great! Would you be open to writing a blogpost about it?

**Reply by aatishg01** [reply]
Sure what would that entail?

**Reply by Bananazooka** [reply]
You would have to write a post on WordPress (min 300 words) explaining what you did and how! You can find examples [here](https://www.bitcraze.io/category/guest-blogger/) of what other people did. If you're interested, just send us an email at contact@bitcraze.io and I'll set everything up!
